command_line/refine.py

```diff
@@ -116,411 +116,452 @@
       .expert_level = 1
   }
 
-  do_scan_varying_macrocycle = False
-    .type = bool
-    .help = "Override whatever scan_varying is set to and force one round of"
-            "static refinement followed by one round of scan-varying refinement"
-    .expert_level = 3
+  n_static_macrocycles = 1
+    .type = int(value_min=1)
+    .help = "Number of macro-cycles of static refinement to perform"
 
   include scope dials.algorithms.refinement.refiner.phil_scope
 """,
     process_includes=True,
 )
 
 # local overrides for refiner.phil_scope
-phil_overrides = parse(
+phil_overrides = libtbx.phil.parse(
     """
   refinement
   {
     verbosity = 2
+    parameterisation.scan_varying = Auto
```

Comment:
The help string should be updated to explain what the Auto option does.

Reply:
Resolved in https://github.com/dials/dials/pull/751/commits/ddcd0dd574ddb6b73868fba88145140805133d27